src/lang-promql/complete/hybrid.ts

```diff
@@ -284,16 +262,20 @@ export class HybridComplete implements CompleteStrategy {
 
     for (const completionList of data) {
       for (const label of completionList.labels) {
-        const completionResult = this.filter(escape(label), text, label, context);
-        if (completionResult) {
+        const completionResult = context.filter(
+          { label: label, original: label, apply: label, type: completionList.type, score: 0 },
+          text
+        );
+        console.log('==', completionResult);
+        if (completionResult !== null) {
           options.push(completionResult);
         }
       }
     }
     if (includeSnippet) {
       for (const s of parsedSnippets) {
-        const completionResult = this.filter(s.label, text, s.apply, context);
-        if (completionResult) {
+        const completionResult = context.filter(s, text);
+        if (completionResult !== null) {
```

Comment:
You really don't like the simple check :
```typescript
if (completionResult) {
}
```

Reply:
Yeah, that's just the wild west to me, that could mean anything ;)

Reply:
Guess it's a question of habits. But ok it's fine